content/rs/clusters/monitoring/uptrace-integration.md

```diff
@@ -0,0 +1,189 @@
+---
+Title: Uptrace integration with Redis Enterprise Software
+linkTitle: Uptrace integration
+description: To collect, display, and monitor metrics data from your databases and other cluster components, you can connect Uptrace to your Redis Enterprise cluster using OpenTelemetry Collector.
+weight: 50
+alwaysopen: false
+categories: ["RS"]
+---
+
+To collect, display, and monitor metrics data from your databases and other cluster components, you can connect Uptrace to your Redis Enterprise cluster using OpenTelemetry Collector.
+
+[Uptrace](https://uptrace.dev/get/) is an open source application performance monitoring (APM) tool that supports distributed tracing, metrics, and logs. You can use it to monitor applications and set up automatic alerts to receive notifications.
+
+With [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/), you can receive, process, and export telemetry data to any monitoring tool. You can use it to scrape Prometheus metrics provided by Redis and then export those metrics to Uptrace.
+
+You can use Uptrace to:
+
+- Collect and display data metrics not available in the [admin console]({{< relref "/rs/clusters/monitoring/console-metrics-definitions.md" >}}).
+- Use prebuilt dashboard templates maintained by the Uptrace community.
+- Set up automatic alerts and receive notifications via email, Slack, Telegram, and others.
+- Monitor your app performance and logs using [tracing](https://uptrace.dev/opentelemetry/distributed-tracing.html).
+
+![uptrace-redis-nodes](/images/rs/uptrace-redis-nodes.png)
+
+## Install Collector and Uptrace
+
+Because installing OpenTelemetry Collector and Uptrace can take some time, you can use the [docker-compose](https://github.com/uptrace/uptrace/tree/master/example/redis-enterprise) example that also comes with Redis Enterprise cluster and Alertmanager.
```

Comment:
It looks like the Uptrace docs use "AlertManager" rather than "Alertmanager".
```suggestion
Because installing OpenTelemetry Collector and Uptrace can take some time, you can use the [docker-compose](https://github.com/uptrace/uptrace/tree/master/example/redis-enterprise) example that also comes with Redis Enterprise cluster and AlertManager.
```